spk/bazarr/src/service-setup.sh

```diff
@@ -0,0 +1,33 @@
+# Package
+PACKAGE="bazarr"
+
+# Others
+PYTHON_DIR="/var/packages/python38/target"
+
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+
+GROUP="sc-download"
+SVC_BACKGROUND=y
+SVC_WRITE_PID=y
+SVC_CWD="${SYNOPKG_PKGDEST}/share/${PACKAGE}"
+
+export SYNOPKG_PKGDEST
+export SYNOPKG_PKGVAR
+
+service_postinst ()
+{
+
+    # Create a Python virtualenv
```

Comment:
I guess this Python 3 setup has been copy-paste in many packages. It may be a good idea to collect it as a common function that package can just invoke. CC @hgy59